src/vs_handler/mod.rs

```diff
@@ -88,6 +90,27 @@ impl PreviewedScript {
 
             self.message_handler_id = Some(id);
         }
+        fn parse_arg(arg: &str) -> Result<(&str, &str)> {
+            arg.find('=')
+               .map(|index| arg.split_at(index))
+               .map(|(k, v)| (k, &v[1..]))
+               .ok_or_else(|| anyhow!("No value specified for argument: {}", arg))
+        }
+
+        // https://github.com/YaLTeR/vapoursynth-rs/blob/60f2656ed76a3f5dc40266618a82d80f7b49fbd6/vapoursynth/examples/vspipe.rs#L722
+        if let Some(args) = &self.args {
+            let mut args_map = OwnedMap::new(API::get().unwrap());
+            for arg in args.iter().map(|s| &**s).map( parse_arg) {
```

Comment:
This mapping seems unnecessary, there exists a [split](https://doc.rust-lang.org/std/primitive.str.html#method.split) function that's more idiomatic.

I'll comment a better version when I have time later.
`parse_arg` could be completely removed and there's no need to add `.iter()` over a `Vec`.